test/unit/trace.test.js

```diff
@@ -351,6 +354,53 @@ describe('Trace', function() {
     expect(events.length).to.equal(0)
   })
 
+  it('parent.* attributes should be present on generated spans', function() {
```

Comment:
In the debates between unit tests and integration tests... this is leaning more on the integration test realm to me. Given the usage of DT mechanisms to trace/span impacts. But many of our unit tests blur this line. As such, won't get too picky.